UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -184,6 +184,14 @@ <h2>Publisher metadata ecosystem</h2>
                             Finally from the "Metadata Processing" layer both the Display Techniques for EPUB and ONIX flow into the "Resulting Statements" layer "User Experience Guide for Displaying Accessibility Metadata"</p>
                     </details>
 
+<h2>Legal Considerations</h2>
+
+<p> in some jurisdictions  publishers may have an exemption from the provision of accessible publications, including the provision of accessibility metadata. Some examples of this are when the publisher claims a disproportionate burden  exemption or when it is a micro-enterprise (e.g., enterprises employing fewer than 10 people and with annual
+turnover or balance sheet total not exceeding Γé¼2 million).</p>
+
+<p >In cases    where the publisher is claiming an exemption it is up to  implementors to decide what to present to end users. Publishers may not want their exemption information to be made public. It may be advisable for implementors to instead provide guidance to the end user by including a user-friendly statement such as, "This title may not yet be fully accessible to all readers. Check the list of accessibility features to determine  if this title is suitable for your way of reading."</p>
```

Comment:
I would delete "it is up to  implementers to decide what to present to end users." to keep the sentence more clear on the fact that "Publishers may not want". 
New proposed sentence: "In cases where the publisher is claiming an exemption, it does not mean that they want their exemption information to be made public.
(again, if I risk that one implementer displays that, then I will not use the metadata)